src/controllers/bookshelf.controller.js

```diff
@@ -0,0 +1,29 @@
+const bookshelfService = require("../services/bookshelf.service");
+
+const getBookshelfByUserId = async (req, res) => {
+  try {
+    const userId = user.id;
```

Comment:
해당 부분으로 userId가 정상적으로 잘 받아지는 지 확인 부탁드립니다~!

```suggestion
    const userId = req.user.id;
```